src/Core/Grid/Definition/Factory/CategoryGridDefinitionFactory.php

```diff
@@ -129,7 +128,7 @@ protected function getColumns()
                     ])
             )
             ->add(
-                (new HtmlColumn('description'))
+                (new DataColumn('description'))
```

Comment:
Why change this? Since the description can contain HTML code will it not be rendered weirdly with HTML tags and all? The advantage is that it will limit the size of displayed content (since the column includes a default limitation via `max_displayed_characters` but the content itself should be stripped from tags before it's sliced no? Which doesn't seem to be done anywhere for now, so it will display partially split HTML content

Reply:
I think it was accidentally changed after merge, since I added HtmlColumn in another PR. 
Fixed.